Ginger/Ginger/SolutionWindows/TreeViewItems/ApplicationModelsTreeItems/AppApiModelTreeItem.cs

```diff
@@ -91,5 +94,33 @@ public void SetTools(ITreeView TV)
             
             AddSourceControlOptions(mContextMenu);
         }
+
+        public override void DuplicateTreeItem(object item)
+        {
+            if (item is RepositoryItemBase)
```

Comment:
No need the IF-Else- because for sure it is a RepositoryItemBase- it's an API model